frontend/ui/widget/radiobuttonwidget.lua

```diff
@@ -1,3 +1,34 @@
+--[[--
+Widget that allows to select an entry from a radiobutton list.
```

Comment:
Something like this maybe?
```suggestion
Widget that allows selecting an entry from a @{widget.radiobutton|RadioButton} list.
```